src/main/java/frc/robot/RobotContainer.java

```diff
@@ -240,76 +240,88 @@ private void configureBindings() {
                 .onFalse(new InstantCommand(() -> endgameSubsystem.setVolts(0.0, 0)));
         }
 
         if (FeatureFlags.runIntake) {
-            controller.a()
+            controller.b()
                 .onTrue(new InstantCommand(
-                        () -> intakeSubsystem.run(IntakeAction.INTAKE)));
-                
-            controller.start()
+                        () -> intakeSubsystem.run(IntakeAction.INTAKE)))
+                .onFalse(new InstantCommand(
+                    () -> intakeSubsystem.run(IntakeAction.NONE)));
+
+            controller.a()
                 .onTrue(new InstantCommand(
-                        () -> intakeSubsystem.run(IntakeAction.NONE)));
+                    () -> intakeSubsystem.run(IntakeAction.REVERSE)))
+                .onFalse(new InstantCommand(
+                    () -> intakeSubsystem.run(IntakeAction.NONE)));
         }
 
         if (FeatureFlags.runScoring) {
-            controller.a()
+            controller.b()
                 .onTrue(new InstantCommand(
                     () -> scoringSubsystem.setAction(
                         ScoringSubsystem.ScoringAction.INTAKE)))
                 .onFalse(new InstantCommand(
                     () -> scoringSubsystem.setAction(
                         ScoringSubsystem.ScoringAction.WAIT)));
 
-            controller.b()
+            controller.a()
+                .onTrue(new InstantCommand(
+                    () -> scoringSubsystem.setAction(
+                        ScoringSubsystem.ScoringAction.SPIT)))
+                .onFalse(new InstantCommand(
+                    () -> scoringSubsystem.setAction(
+                        ScoringSubsystem.ScoringAction.WAIT)));
+
+            controller.povUp()
                 .onTrue(new InstantCommand(
                     () -> scoringSubsystem.setAction(
                          ScoringSubsystem.ScoringAction.AIM)));
 
-            controller.x()
+            controller.rightBumper()
                 .onTrue(new InstantCommand(
                     () -> scoringSubsystem.setAction(
                         ScoringSubsystem.ScoringAction.SHOOT)));
 
-            controller.y()
+            controller.povLeft()
                 .onTrue(new InstantCommand(
                     () -> scoringSubsystem.setAction(
                         ScoringSubsystem.ScoringAction.ENDGAME)));
 
-            controller.back()
+            controller.povRight()
                 .onTrue(new InstantCommand(
                     () -> scoringSubsystem.setAction(
                         ScoringSubsystem.ScoringAction.AMP_AIM)));
 
-            controller.start()
+            controller.povDown()
                 .onTrue(new InstantCommand(
                     () -> scoringSubsystem.setAction(
                         ScoringSubsystem.ScoringAction.WAIT)));
 
-            controller.povLeft()
+            controller.y()
```

Comment:
We should use POV left to trigger auto align to source, and then use a shooter intake button to do the source intake action